nvchecker/source/gitlab.py

```diff
@@ -51,3 +58,16 @@
   else:
     version = data[0]['created_at'].split('T', 1)[0].replace('-', '')
   return version
+
+def check_ratelimit(exc, name):
+  res = exc.response
+  n = int(res.headers.get('RateLimit-Remaining'))
+  if n == 0:
+    reset = int(res.headers.get('RateLimit-Reset'))
+    logger.error('rate limited, resetting at %s. '
+                 'Or get an API token to increase the allowance if not yet'
+                 % time.ctime(reset),
+                 name = name,
+                 reset = reset)
```

Comment:
We can't get the real reset time. Let's put `(unknown)` there instead.

Reply:
Fixed.